File: subprojects/frida-node/examples/open_service/plist.ts
import frida from "frida";

async function main() {
    const device = await frida.getUsbDevice();

    const diag = await device.openService("plist:com.apple.mobile.diagnostics_relay");
    await diag.request({ type: "query", payload: { Request: "Sleep", WaitForDisconnect: true } });
    await diag.request({ type: "query", payload: { Request: "Goodbye" } });
}

main()
    .catch(e => {
        console.error(e);
    });

File: subprojects/frida-python/examples/open_service/plist.py
import frida

device = frida.get_usb_device()

diag = device.open_service("plist:com.apple.mobile.diagnostics_relay")
diag.request({"type": "query", "payload": {"Request": "Sleep", "WaitForDisconnect": True}})
diag.request({"type": "query", "payload": {"Request": "Goodbye"}})
